src/posix/platform/radio_spinel.cpp

```diff
@@ -407,6 +407,29 @@ void RadioSpinel::Init(const char *aRadioFile, const char *aRadioConfig)
     SuccessOrExit(error = Get(SPINEL_PROP_HWADDR, SPINEL_DATATYPE_UINT64_S, &gNodeId));
     gNodeId = ot::Encoding::BigEndian::HostSwap64(gNodeId);
 
+    {
+        unsigned int   versionMajor;
+        unsigned int   versionMinor;
+        uint8_t        data[2 * (sizeof(uint32_t) + 1)];
+        unsigned int   len = sizeof(data);
+        spinel_ssize_t parsedLen;
+
+        SuccessOrExit(error = Get(SPINEL_PROP_PROTOCOL_VERSION, SPINEL_DATATYPE_DATA_S, data, &len));
```

Comment:
Is it possible to use `Get(SPINEL_PROP_PROTOCOL_VERSION, SPINEL_DATATYPE_UINT_PACKED_S SPINEL_DATATYPE_UINT_PACKED_S, &versionMajor, &versionMinor)`?

Reply:
Yes... Thanks for the hint. 
For some reason I thought we can only get one value type using `Get()`.